virtio-devices/src/net.rs

```diff
@@ -162,6 +162,8 @@ pub enum Error {
     TapError(TapError),
     #[error("Error calling dup() on tap fd: {0}")]
     DuplicateTapFd(std::io::Error),
+    #[error("No tap device, at least one should be created")]
```

Comment:
This isn't about taps provided by the user. This is a tap device missing when creating the network.  If there is no reason a user can trigger it then it should be an assert! i.e. programming error.

Reply:
Sounds good, I'll use an `assert!()`.